proxysql-admin

```diff
@@ -2063,6 +2076,20 @@ function update_cluster()
     exit 1
   fi
 
+  # Update mysql server version details in proxysql db 
```

Comment:
The server used by mysql_exec may not be online at this point.  This check needs to be done further down in the code where we have determined the list of nodes.